Extension/pages/skin/my.css

```diff
@@ -273,6 +273,7 @@ a.spl-user-add-spn {
 }
 
 .popup-footer span {
+    overflow: hidden;

```

Comment:
So what? It just breaks the line, overflow hidden does nothing:
!(http://content.screencast.com/users/ameshkov/folders/Jing/media/26211883-a695-491a-a52b-2a843363589b/00000913.png)